### ISSUE TITLE
[Feat] r.import does not create imagery groups

### ISSUE THREAD

**Issue by marisn** [OSGeo/grass#550]
**Is your feature request related to a problem? Please describe.**
When importing a multichannel image with r.in.gdal, it automatically creates an imagery group containing all imported rasters. When importing the same image with r.import, no such group is created.

**Describe the solution you'd like**
r.import should match r.in.gdal behaviour and create imagery groups.

**Describe alternatives you've considered**
It is still possible to create imagery groups with i.group afterwards.

**Additional context**
Tested with current master.

**Comment by metzm** [comment]
Initially, r.import was not designed to be a full replacement of r.in.gdal + r.proj. Instead, the purpose was to provide easy import for simple raster data. If r.import does not provide the expected results, r.in.gdal an r.proj need to be used.
Considering changes in PROJ where different transformation pipelines might be available to transform coordinates from one CRS to another CRS, users can make an informed decision about which pipeline to use with r.proj.
In short, if r.import does not provide the expected results, use r.in.gdal + r.proj.
